pilot/pkg/features/pilot.go

```diff
@@ -305,6 +305,12 @@ var (
 	XDSAuth = env.RegisterBoolVar("XDS_AUTH", true,
 		"If true, will authenticate XDS clients.").Get()
 
+	EnableXDSAuthorization = env.RegisterBoolVar(
+		"PILOT_ENABLE_XDS_AUTHORIZATION",
```

Comment:
Narrower term ? I'm not comfortable with broad things - authorization can mean a lot of things.
Not a big deal probably, but if you can think of a more specific term... My English vocabulary is probably not large enough.